src/task_8_2.py

```diff
@@ -0,0 +1,20 @@
+# Даны три слова. Выяснить, является ли хоть одно из них палиндромом
+# ("перевертышем"), т. е. таким, которое читается одинаково слева направо и
+# справа налево. (Определить функцию, позволяющую распознавать слова
+# палиндромы.)
+def palindrom(spis: list) -> list:
+    """
+    Функция проверяет наличие слов-палиндромов в списке,
+    и если находит их, выводит его на экран и заносит в новый список
+    """
+    spis_polindromov = []
+    for i in spis:
+        per_i = i[::-1]
+        if i == per_i:
+            spis_polindromov.append(i)
+            print(f"{i} - палиндром")
+    return spis_polindromov
```

Comment:
Каждый палиндром будет выводиться в итоговом print(is_paindrom(spis))
Тут предлагаю использовать генератор списка. What do you think?

```suggestion
    return [i for i in spis if i == i[::-1] ]
```